plop-templates/component/docs/overview.mdx.hbs

```diff
@@ -5,8 +5,8 @@ group: {{projectCategory}}
 storybookPath: /story/{{projectCategory}}-{{componentName}}--basic
 unreleased: true
 figmaGalleryNodeId: REPLACE_ME
-relatedComponents: ['slug-of-component', 'another-slug']
-relatedPatterns: ['slug-of-pattern', 'another-slug']
+# relatedComponents: ['slug-of-component', 'another-slug']
```

Comment:
Commenting these out so new docs pages don't fail when you plop create a new component.